src/test/java/org/reaktivity/nukleus/http2/internal/streams/rfc7540/server/ConnectionManagementIT.java

```diff
@@ -330,4 +331,17 @@ public void pushResourcesWithOverrideHeader() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configure(name = "nukleus.http2.max.cleanup.streams", value = "1")
+    @Configure(name = "nukleus.http2.streams.cleanup.delay", value = "10")
```

Comment:
Please convert these hardcoded strings to constants and add a test to confirm static string name matches property name method return value, so we can catch divergence if the property is renamed in future - same as other nuklei requiring the same type of property override for specific tests via `@Configure` annotation.